infrastructure/src/main/java/org/corfudb/infrastructure/log/Checksum.java

```diff
@@ -0,0 +1,29 @@
+package org.corfudb.infrastructure.log;
+
+import com.google.common.hash.Hasher;
+import com.google.common.hash.Hashing;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class Checksum {
```

Comment:
nit:

1. Can this be refactored to this location? https://github.com/CorfuDB/CorfuDB/blob/4955f4e439c22a2966176a61c530786fa438fc62/infrastructure/src/main/java/org/corfudb/infrastructure/utils
3. And can we rename this class to `ChecksumUtils`?

Reply:
Addressed in https://github.com/CorfuDB/CorfuDB/pull/3532

Btw this patch is dependent on #3532 , so the diff is showing code from the other PR.